smallab/runner.py

```diff
@@ -86,7 +91,7 @@ def _find_uncompleted_specifications(self, name, specifications):
 
     def run(self, name: typing.AnyStr, specifications: typing.List[Specification], experiment: BaseExperiment,
             continue_from_last_run=True, propagate_exceptions=False,
-            force_pickle=False, specification_runner: AbstractRunner = JoblibRunner(None)) -> typing.NoReturn:
+            force_pickle=False, specification_runner: AbstractRunner = JoblibRunner(None),use_dashboard=True) -> typing.NoReturn:
```

Comment:
Also, I didn't catch this before, but you're instantiating `JoblibRunner` as a default arg for `specification_runner`. Default args are only evaluated once, so every instance of `ExperimentRunner` using the default arg for `specification_runner` will have the same instance of `JoblibRunner` - probably not what you want.

A better paradigm for this would be:
```
def run(self, specification_runner: AbstractRunner=None, ...):
    runner = JoblibRunner(None) if specification_runner is None else specification_runner
    ...
```

Reply:
ah - good point. i don't think the runners have any state but probably still good to destroy them. in the future we may want to look at the multiprocessing implementation and reuse processes instead of spawning more if the runner is invoked twice in the same project